bluebrain/documentation/setup_personal.md

```diff
@@ -25,56 +34,70 @@ conjunction with Apple's CLang:
 
     $ brew install gcc
 
+### Building software on Ubuntu
+
+Ubuntu / Debian have a habit of being somewhat _special_, patching upstream
+projects in unexpected ways.
+Consider yourself warned to not rely on system packages and defer to
+Spack-installed ones when needed.
+
+First, ensure that the essential packages to building stuff are installed:
+
+    $ sudo add-apt-repository ppa:ubuntu-toolchain-r/test
+    $ sudo apt update
+    $ sudo apt install build-essential gcc-11 g++-11 gfortran-11
+
+If Python 3 is not your default, tell Ubuntu to use a newer one, e.g.,
+Python 3.8 by setting the default `python`:
+
+    $ sudo update-alternatives --install /usr/bin/python python /usr/bin/python3.8 1
+
+To check that we are using Python 3 as `python`:
+
+    $ sudo update-alternatives --config python
+    There is only one alternative in link group python
+    (providing /usr/bin/python): /usr/bin/python3.8. Nothing to configure.
+
+## Spack Setup
+
 Now clone our version of Spack and find compilers and external packages:
 
     $ git clone -c feature.manyFiles=true https://github.com/BlueBrain/spack.git
     $ . spack/share/spack/setup-env.sh
     $ spack compiler find
     $ spack external find
 
-Edit the resulting externals, removing any references to `brew`, `python`,
-and `sqlite` from the system, the latter two as they are unfit to be used
-as full dependencies with Spack:
+### Additional Configuration
 
-    $ spack config edit packages
+#### Remove Flawed External Packages
 
-With this minimal setup, Spack should operate independent of the system and
-the `brew` installation.
-Software installed via Spack should be accessed either with `spack load` or
-by using Spack's environment feature.
+Use the following commands to remove software that may be too customized to
+use reliably:
 
-## Building software on Ubuntu
+    $ spack config rm packages:hdf5
+    $ spack config rm packages:openmpi
+    $ spack config rm packages:python
+    $ spack config rm packages:sqlite
 
-Use Spack on the workstations provided by the project.
+#### When Building Steps
 
-### Ubuntu 18.04
+Use the following commands to skip building the FLTK GUI interface for
+GMSH, as it pulls in too many dependencies:
 
-We build Docker images based on Ubuntu 18.04, and the same settings can be
-used to set Spack up on the desktops:
+    $ spack config add packages:gmsh:variants:"~mmg~fltk"
```

Comment:
I am adding a doc in steps. This is not needed

Reply:
I do not understand why I cannot give a suggestion here

Reply:
You should be able to!

I can take this out.

Reply:
Done. Thank you for testing and reviewing!